lib/parse/datetime/parser.ex

```diff
@@ -318,7 +318,16 @@ defmodule Timex.Parse.DateTime.Parser do
       hour when hour in [:hour24, :hour12] ->
         %{date | :hour => value}
       :min       -> %{date | :minute => value}
-      :sec       -> %{date | :second => value}
+      :sec       ->
+        case value do
+          60 ->
+            date
+            |> Timex.to_naive_datetime()
+            |> (&%{&1 | second: 0}).()
```

Comment:
I would prefer this line to be either `Map.put(:second, 0)` or `Timex.set(second: 0)`, as it is clearer what is happening. Either that or break up the pipeline. For readability, I prefer to never use anonymous functions in pipelines, especially the shorthand syntax, since it is unnecessarily noisy compared to `fn ... -> ... end`.

Other than that, I think this looks good, and can be merged once that change is made.

Reply:
Done!

Reply:
@bitwalker can you merge this please?